kubernetes/check.py

```diff
@@ -55,6 +55,13 @@
 
 EVENT_TYPE = 'kubernetes'
 
+# Mapping between k8s events and ddog alert types per
+# https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/record/event.go#L60
```

Comment:
This link is guaranteed to break, could you point to a permalink instead?

Reply:
This is the link to the commit that introduced the change https://github.com/kubernetes/kubernetes/blob/adb75e1fd17b11e6a0256a4984ef9b18957d94ce/staging/src/k8s.io/client-go/1.4/tools/record/event.go#L59